package.json

```diff
@@ -90,7 +85,8 @@
     "redux": "4.0.4",
     "redux-logger": "3.0.6",
     "redux-persist": "6.0.0",
-    "rootstrap-ui": "0.0.2"
+    "rootstrap-ui": "0.0.2",
```

Comment:
probably can remove this package too right?

Reply:
yes! Removed, and also finished with cypress removing, I thought I had already deleted the entire folder 😅 nice catch!